Documentation/dev-guide/interacting_v3.md

```diff
@@ -6,7 +6,7 @@ Users mostly interact with etcd by putting or getting the value of a key. This s
 
 The API version used by etcdctl to speak to etcd may be set to version `2` or `3` via the `ETCDCTL_API` environment variable. By default, etcdctl on master (3.4) uses the v3 API and earlier versions (3.3 and earlier) default to the v2 API.
 
-Note that any key that was created using the v2 API will not be able to be queried via the v2 API.  A v3 API ```etcdctl get``` of a v2 key will exit with 0 and no key data, this is the expected behaviour.
+Note that any key that was created using the v2 API will not be able to be queried via the v3 API.  A v3 API ```etcdctl get``` of a v2 key will exit with 0 and no key data, this is the expected behaviour.
```

Comment:
Good catch, but you need to update your commit title to pass the build, similar to the PR title, in this format `Expected commit title format '<package>{", "<package>}: <description>'` Thanks!